commands/scan/jasrunner_cves.go

```diff
@@ -0,0 +1,44 @@
+package scan
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-cli-security/commands/audit/jas"
+	"github.com/jfrog/jfrog-cli-security/commands/audit/jas/applicability"
+	"github.com/jfrog/jfrog-cli-security/commands/audit/jas/secrets"
+
+	"github.com/jfrog/jfrog-cli-security/utils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+)
+
+func runJasScannersAndSetResults(scanResults *utils.Results, cveList []string,
+	serverDetails *config.ServerDetails, workingDirs []string) (err error) {
+
+	if serverDetails == nil || len(serverDetails.Url) == 0 {
+		log.Warn("To include 'Advanced Security' scan as part of the audit output, please run the 'jf c add' command before running this command.")
+		return
+	}
+	scanner, err := jas.NewJasScanner(workingDirs, serverDetails)
+	if err != nil {
+		return
+	}
+
+	defer func() {
+		cleanup := scanner.ScannerDirCleanupFunc
+		err = errors.Join(err, cleanup())
+	}()
+
+	scanResults.ExtendedScanResults.ApplicabilityScanResults, err = applicability.RunApplicabilityWithScanCves(scanResults.GetScaScansXrayResults(), cveList, scanResults.GetScaScannedTechnologies(), scanner)
+	if err != nil {
+		fmt.Println("there was an error:", err)
+		return
+	}
+
+	scanResults.ExtendedScanResults.SecretsScanResults, err = secrets.RunSecretsScan(scanner, secrets.SecretsScannerDockerScanType)
+	if err != nil {
+		return
+	}
+	return
+}
```

Comment:
You already have `jasrunner.go` implementing this functionality, modify it to run only what you need.
You can use a Param struct to pass the needed arguments for each type.

Reply:
Done